act-rules-format/act-rules-format.bs

```diff
@@ -433,6 +435,42 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </aside>
 
+<aside class=example>
+  <header>Example subjective applicability for a rule testing that elements styled as a heading use correct heading markup. This rule applicability cannot be written objectively since "styled as a heading" is subjective and depends on the context of the page.</header>
+  <blockquote>
+    <p>The rule applies to any HTML element that is styled as a heading.</p>
+  </blockquote>
+</aside>
+
+<aside class=example>
+  <header>Example subjective applicability for a rule testing that error messages correctly describe an invalid form field value. This rule applicability cannot be written objectively since what constitutes a "form field error indicator" is subjective.</header>
+  <blockquote>
+    <p>Applicability: This rule applies to all form field error indicators.</p>
+    <p>Expectation 1: This form field error indicator allows the identification of the related form field.</p>
+    <p>Expectation 2: The form field error indicator describes the cause of the error or how to solve it.</p>
+  </blockquote>
+</aside>
+
+<aside class=example>
+  <header><strong>Incorrect Example:</strong>This example demonstrates an applicability that attempts to be subjective, but is so broad that it does not meet the "must be unambiguous" requirement. For example, would a text emoji like ":)" be considered non-text content that requires an accessible name? Due to the ambiguity in the applicability it is impossible to tell.</header>
```

Comment:
Possibly include that a rule with this applicability is not an atomic rule and should be disallowed.

Reply:
Consider an appendix with more extensive examples such as this where more discussion/commentary is needed to describe why the example is good/bad.